lib/timeline/timeline_protocol.dart

```diff
@@ -117,6 +117,9 @@ class TimelineThread implements Comparable<TimelineThread> {
 
   TimelineThreadEvent durationStack;
 
+  bool get isDartThread => _name == 'io.flutter.1.ui';
```

Comment:
wow those names look a bit magic. Can you add a comment explaining where they are from.
This would also be a good case to have a test so that if flutter ever changes the magic names we catch the breakage immediately on our build bots.

Reply:
Perhaps it would be better to check what the names end with instead. This is how thread priority is selected

`int get sortPriority {
    if (name.endsWith('.ui')) {
      return 1;
    }
    if (name.endsWith('.gpu')) {
      return 2;
    }
    if (name.startsWith('io.flutter.')) {
      return 3;
    }
    return 4;
  }`